feed/routes.py

```diff
@@ -0,0 +1,49 @@
+"""URL routes for RSS feeds."""
+
+from werkzeug import Response
+from werkzeug.exceptions import BadRequest
+from flask import Blueprint, request, make_response
+
+from feed import controller
+from feed.errors import FeedError
+from feed.consts import FeedVersion
+
+blueprint = Blueprint("rss", __name__, url_prefix="/rss")
+
+
+# FIXME: I would change this in the following way:
+#        Root url / is predefined atom or rss 2.0 (I have no preference), and
```

Comment:
I agree. I think the `url_prefix` for the blueprint should just be `"/"`.  RSS is still the dominant format even if Atom is more robust, so RSS as default is fine. There can be separate routes for `/rss.xml` and `/atom.xml`.

Reply:
Added but which version do you like better:

```
/rss/cs.LO       -> RSS 2.0
/atom/cs.LO      -> Atom 1.0
/cs.LO           -> RSS 2.0
```
or

```
/rss.xml?q=cs.LO
/atom.xml?q=cs.LO
/q=cs.LO
```

And what should be the default behavior if the query is not provided? Error?